staging/scripts/destroy-disk

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
This is meant to help with deleting disks without remembering or copying/pasting weird commands